intel_nn_hal/PreparedModel.cpp

```diff
@@ -395,22 +409,24 @@ bool PreparedModel::initializeRunTimeOperandInfo() {
                 to.type = from.type;
                 break;
             case OperandType::TENSOR_QUANT8_SYMM:
-                    to.type = from.type;
-                    break;
+                to.type = from.type;
+                break;
             case OperandType::TENSOR_QUANT8_ASYMM_SIGNED:
                     to.type = from.type;
             break;
             case OperandType::TENSOR_QUANT16_SYMM:
                     to.type = from.type;
             break;
             case OperandType::FLOAT16:
+            case OperandType::TENSOR_FLOAT16:
                     to.type = from.type;
             break;
             case OperandType::TENSOR_QUANT8_ASYMM:
-                ALOGE("OperandType::TENSOR_QUANT8_ASYMM is not supported");
+                    to.type = from.type;
+                    ALOGE("OperandType::TENSOR_QUANT8_ASYMM is  supported");
```

Comment:
@anikulk Is the comment supposed to be `is supported` or `is not supported`? Remove the extra space if its the former.